# NLP4Science Papers

Must-read papers on NLP for science (protein and molecular). The paper list is mainly mantained by team [ZJUNLP](https://github.com/orgs/zjunlp/repositories)

## Contents

- [Papers](#papers)
  - [Contents](#contents)
  - [Introduction](#introduction)
  - [Papers](#papers)
    - [Nature and its series ](#nature)
    - [Other Journals and Conferences](#Others)
  - [Contributors](#other-contributors)




## NLP for Protein Papers

### Nature and its series  

1. **Large language models generate functional protein sequences across diverse families.**  Nature Biotechnology. ![](https://img.shields.io/badge/68.164-blue) ![](https://img.shields.io/badge/PLM_models-red) *Ali Madani, Ben Krause, Eric R. Greene, Subu Subramanian, Benjamin P. Mohr, James M. Holton, Jose Luis Olmos Jr., Caiming Xiong, Zachary Z. Sun, Richard Socher, James S. Fraser & Nikhil Naik.* [[pdf](https://www.nature.com/articles/s41587-022-01618-2)].  2023.1.
2. **I-TASSER-MTD: a deep-learning-based platform for multi-domain protein structure and function  prediction.**  Nature Protocols. ![](https://img.shields.io/badge/13.491-blue) ![](https://img.shields.io/badge/Structure-red) *Xiaogen Zhou, Wei Zheng, Yang Li, Robin Pearce, Chengxin Zhang, Eric W. Bell, Guijun Zhang & Yang Zhang.* [[pdf](https://www.nature.com/articles/s41596-022-00728-0)].  2022.8. 
3. **SANA: cross-species prediction of Gene Ontology GO annotations via topological network alignment.**  npj | Systems Biology and Applications. ![](https://img.shields.io/badge/4.333-blue) ![](https://img.shields.io/badge/PPI-red) *Siyue Wang, Giles R. S. Atkinson & Wayne B. Hayes.* [[pdf](https://www.nature.com/articles/s41540-022-00232-x)].  2022.7. 
4. **Learning functional properties of proteins with language models.**  Nature Machine Intelligence. ![](https://img.shields.io/badge/16.649-blue) ![](https://img.shields.io/badge/PLM_models-red) *Serbulent Unsal, Heval Atas, Muammer Albayrak, Kemal Turhan, Aybar C. Acar & Tunca Doğan* [[pdf](https://www.nature.com/articles/s41540-022-00232-x)].  2022.3. 
5. **Protein function prediction for newly sequenced organisms.**  Nature Machine Intelligence. ![](https://img.shields.io/badge/16.649-blue) ![](https://img.shields.io/badge/label_propagation-red) Mateo Torres, Haixuan Yang, Alfonso E. Romero & Alberto Paccanaro [[pdf](https://www.nature.com/articles/s42256-021-00419-7)].  2021.12. 
6. **Structure-based protein function prediction using graph convolutional networks**  Nature Communications. ![](https://img.shields.io/badge/14.919-blue) ![](https://img.shields.io/badge/Structure-red) *Vladimir Gligorijević, P. Douglas Renfrew, Tomasz Kosciolek, Julia Koehler Leman, Daniel Berenberg, Tommi Vatanen, Chris Chandler, Bryn C. Taylor, Ian M. Fisk, Hera Vlamakis, Ramnik J. Xavier, Rob Knight, Kyunghyun Cho & Richard Bonneau.* [[pdf](https://www.nature.com/articles/s41467-021-23303-9)].  2021.4. 
7. **Expanding functional protein sequence spaces using generative adversarial networks**  Nature Machine Intelligence. ![](https://img.shields.io/badge/16.649-blue) ![](https://img.shields.io/badge/GAN-red) Donatas Repecka, Vykintas Jauniskis, Laurynas Karpus, Elzbieta Rembeza, Irmantas Rokaitis, Jan Zrimec, Simona Poviloniene, Audrius Laurynenas, Sandra Viknander, Wissam Abuajwa, Otto Savolainen, Rolandas Meskys, Martin K. M. Engqvist & Aleksej Zelezniak. [[pdf](https://www.nature.com/articles/s42256-021-00310-5)].  2021.3.
8. **Protein function prediction is improved by creating synthetic feature samples with generative adversarial networks**  Nature Machine Intelligence. ![](https://img.shields.io/badge/16.649-blue) ![](https://img.shields.io/badge/synthetic_feature-red) *Cen Wan & David T. Jones.* [[pdf](https://www.nature.com/articles/s42256-020-0222-1)].  2020.8

### Other Journals and Conferences

1. **Exploring Evolution-aware & free protein language models as protein function predictors.**  NeurIPS 2022. ![](https://img.shields.io/badge/PLMs-red) *Hu, Mingyang and Yuan, Fajie and Yang, Kevin K and Ju, Fusong and Su, Jin and Wang, Hui and Yang, Fei and Ding, Qiuyang*  [[pdf](https://openreview.net/pdf?id=U8k0QaBgXS)], 2022.06
2. **A Semi-supervised Graph Deep Neural Network for Automatic Protein Function Annotation.** Bioinformatics and Biomedical Engineering. ![](https://img.shields.io/badge/GNN-red) *Akrem Sellami, Bishnu Sarker, Salvatore Tabbone, Marie-Dominique Devignes & Sabeur Aridhi.* [[pdf](https://link.springer.com/chapter/10.1007/978-3-031-07802-6_14)], 2022.6
3. **ProTranslator: zero-shot protein function prediction using textual description.** arXiv:2204.10286.   ![](https://img.shields.io/badge/textual-red) *Hanwen Xu, Sheng Wang.* [[pdf](https://arxiv.org/abs/2204.10286)], 2022.5
4. **A deep learning framework for predicting protein functions with co-occurrence of GO terms.** IEEE/ACM Trans Comput Biol Bioinform.  ![](https://img.shields.io/badge/co_occurrence-red) *Min Li, Wenbo Shi, Fuhao Zhang, Min Zeng, Yaohang Li.*  [[pdf](https://pubmed.ncbi.nlm.nih.gov/35476573/)], 2022.5
5. **PANDA2: protein function prediction using graph neural networks.** NAR Genomics and Bioinformatics. ![](https://img.shields.io/badge/GNN-red) *Chenguang Zhao,  Tong Liu,  Zheng Wang.*  [[pdf](https://academic.oup.com/nargab/article/4/1/lqac004/6520105)], 2022.2
6. **DeepGraphGO: graph neural network for large-scale, multispecies protein function prediction.** Bioinformatics.  ![](https://img.shields.io/badge/GNN-red) *Ronghui You,  Shuwei Yao,  Hiroshi Mamitsuka,  Shanfeng Zhu.*  [[pdf](https://academic.oup.com/bioinformatics/article/37/Supplement_1/i262/6319663)], 2021.7
7. **Accurate Protein Function Prediction via Graph Attention Networks with Predicted Structure Information.**  *Briefings in Bioinformatics*. ![](https://img.shields.io/badge/GAN-red) *Boqiao Lai,  Jinbo Xu.*  [[pdf](https://www.biorxiv.org/content/10.1101/2021.06.16.448727v1)], 2021.6
8. **Effusion: prediction of protein function from sequence similarity networks.** Bioinformatics.  ![](https://img.shields.io/badge/PPI-red) *Jeffrey M Yunes,  Patricia C Babbitt.*  [[pdf](https://academic.oup.com/bioinformatics/article/35/3/442/5063404)], 2019.2
9. **Protein function prediction from dynamic protein interaction network using gene expression data.** Journal of Bioinformatics and Computational Biology. . ![](https://img.shields.io/badge/gene_expression-red) *Sovan Saha, Abhimanyu Prasad, Piyali Chatterjee, Subhadip Basu and Mita Nasipuri.*  [[pdf](https://www.worldscientific.com/doi/pdf/10.1142/S0219720019500252)], 2018.12
10. **DeepText2GO: Improving large-scale protein function prediction with deep semantic text representation.**  Methods. ![](https://img.shields.io/badge/semantic_text-red) *Ronghui You, Xiaodi Huang, Shanfeng Zhu.*  [[pdf](https://pubmed.ncbi.nlm.nih.gov/29883746/)], 2018.08

## NLP for Molecular Papers

### Nature and its series 

1. **A deep-learning system bridging molecule structure and biomedical text with comprehension comparable to human professionals.**  Nature Communications. ![](https://img.shields.io/badge/17.694-blue) ![](https://img.shields.io/badge/PLM_models-red) *Zheni Zeng, Yuan Yao, Zhiyuan Liu, Maosong Sun.* [[pdf](https://www.nature.com/articles/s41467-022-28494-3)].  2022.02
2. **Language models can learn complex molecular distributions.**  Nature Communications. ![](https://img.shields.io/badge/17.694-blue) ![](https://img.shields.io/badge/PLM_models-red) *Daniel Flam-Shepherd, Kevin Zhu, Alán Aspuru-Guzik.* [[pdf](https://www.nature.com/articles/s41467-022-30839-x)].  2022.07
3. **Large-scale chemical language representations capture molecular structure and properties.** Nature Machine Intelligence. ![](https://img.shields.io/badge/17.694-blue) ![](https://img.shields.io/badge/PLM_models-red) *Jerret Ross, Brian Belgodere, Vijil Chenthamarakshan, Inkit Padhi, Youssef Mroueh, Payel Das.* [[pdf](https://www.nature.com/articles/s42256-022-00580-7)].  2022.12
4. **Leveraging molecular structure and bioactivity with chemical language models for de novo drug design.** Nature Communications. ![](https://img.shields.io/badge/17.694-blue) ![](https://img.shields.io/badge/PLM_models-red) *Michael Moret, Irene Pachon Angona, Leandro Cotos, Shen Yan, Kenneth Atz, Cyrill Brunner, Martin Baumgartner, Francesca Grisoni, Gisbert Schneider.* [[pdf](https://www.nature.com/articles/s41467-022-35692-6)].  2023.01


### Other Journals and Conferences

1. **Mol-Instructions: A Large-Scale Biomolecular Instruction Dataset for Large Language Models.** Arxiv. ![](https://img.shields.io/badge/LLM-red) *Yin Fang, Xiaozhuan Liang, Ningyu Zhang, Kangwei Liu, Rui Huang, Zhuo Chen, Xiaohui Fan, Huajun Chen.* [[pdf](https://arxiv.org/abs/2306.08018)], 2023
2. **Domain-Agnostic Molecular Generation with Self-feedback.** Arxiv. ![](https://img.shields.io/badge/PLM_models-red) *Yin Fang, Ningyu Zhang, Zhuo Chen, Xiaohui Fan, Huajun Chen.* [[pdf](https://arxiv.org/abs/2301.11259)], 2023
3. **Empowering Molecule Discovery for Molecule-Caption Translation with Large Language Models: A ChatGPT Perspective.** Arxiv. ![](https://img.shields.io/badge/PLM_models-red) *Jiatong Li, Yunqing Liu, Wenqi Fan, Xiao-Yong Wei, Hui Liu, Jiliang Tang, Qing Li.* [[pdf](https://arxiv.org/abs/2306.06615)], 2023
4. **Translation between Molecules and Natural Language.**  EMNLP. ![](https://img.shields.io/badge/PLM_models-red) *Carl Edwards, Tuan Lai, Kevin Ros, Garrett Honke, Kyunghyun Cho, Heng Ji.*  [[pdf](https://arxiv.org/pdf/2204.11817.pdf)], 2022
5. **Chemformer: a pre-trained transformer for computational chemistry.**  Machine Learning. ![](https://img.shields.io/badge/PLM_models-red) *Ross Irwin, Spyridon Dimitriadis, Jiazhen He and Esben Jannik Bjerrum.* [[pdf](https://iopscience.iop.org/article/10.1088/2632-2153/ac3ffb)], 2022.01
6. **Multilingual Molecular Representation Learning via Contrastive Pre-training.** ACL. ![](https://img.shields.io/badge/PLM_models-red) *Zhihui Guo, Pramod Sharma, Andy Martinez, Liang Du, Robin Abraham.* [[pdf](https://aclanthology.org/2022.acl-long.242/)], 2022



## Contributors

 We thank all members in team [ZJUNLP](https://github.com/orgs/zjunlp/repositories) for the paper recommendation. Pull requests and issues are welcomed!
